.evergreen/config.yml

```diff
@@ -1687,6 +1687,7 @@ tasks:
         - func: "run tests"
           vars:
             TEST_DATA_LAKE: "true"
+            PYMONGO_MUST_CONNECT: "true"
```

Comment:
This is an EVG var but does it get exported as a shell env var?

Reply:
Good point, I've made it so if PYMONGO_MUST_CONNECT is set in EG it will get pulled explicitly by test-eg.